test/spec/services/apiServiceSpec.js

```diff
@@ -234,6 +235,96 @@ describe("APIService", function(){
       });
     }));
   });
-  */
+
+
+  describe('#availableKinds', function() {
+    var bothSample = ['Binding','ConfigMap','DeploymentConfig','Event','LimitRange','Pod','ReplicaSet','Role','Service', 'Template'];
+    var onlyClusterSample = ['ClusterResourceQuota','Namespace','OAuthAccessToken','PersistentVolume','ProjectRequest','User'];
+
+    // at minimum, prove with tests that it is deduplicating by ignoring openshift!
+    it('should return list of kinds that are scoped to a namespace by default', function() {
+      var namespacedKinds = _.map(APIService.availableKinds(), 'kind');
+      // from the fixtures included, we should have 51 namespaced kinds
+      expect( _.difference(bothSample, namespacedKinds).length ).toEqual(0);
```

Comment:
this doesnt validate that cluster kinds are not included, you need two expectations.  So you need a second line here that checks the difference between cluster sample and all kinds is not 0